### ISSUE TITLE
Starting on new HA Install

### ISSUE THREAD

**Issue by justin654** [mrlt8/docker-wyze-bridge#664]
I am receiving an error when trying to start this in a new HA install on M1 Mac using GenericAArch64. 

"[FATAL tini (7)] exec flask failed: Exec format error"

**Comment by WowSuchRicky** [comment]
Exec format error means you are running a binary for the wrong architecture. You mentioned you're on an M1 Mac which runs arm64 (aarch64) as opposed to x86. 

You may have some luck adding `--platform linux/amd64` to the Docker command line you're running someplace to make sure the containers run properly.

**Comment by mrlt8** [comment]
There seems to be an issue when running the container within Home Assistant #529.

**Comment by justin654** [comment]
> 

Yep, that's exactly what I was doing, running in HA, installed from the add on store.
